README.md

```diff
@@ -9,4 +9,23 @@ Using TDD write Human vs Human game of Tic Tac Toe in java
 - 3x3 only
 - Console
 
-If you get stuck for more than an half-hour on how to write a test, ask for help.
+
+# Running the application
+
+1. Clone the repo {code} git@github.com:gemcfadyen/Apprenticeship-JavaTicTacToe.git {code}
+
+1. Ensure you have JUnit4 on your classpath (which should contain hamcrest-core-1.3.jar, hamcrest-library-1.3.jar).
+
+1. There are no external dependencies or build framework needed.
+
+1. To compile the code outside of an IDE:
+
+   _cd Apprenticeship-JavaTicTacToe/src/main/java/ttt_
+
+   _javac PlayerSymbol.java Cell.java Board.java Prompt.java Player.javaReadFromPromptException.java WriteToPromptException.java HumanPlayer.java CommandPrompt.java Board.java Game.java_
```

Comment:
Use the ` when you are showing snippets of code (shell commands included).

Given that you are compiling everything you can just run `javac *`